cloudfoundry/resource_cf_route_service_binding.go

```diff
@@ -79,13 +107,18 @@ func resourceRouteServiceBindingRead(ctx context.Context, d *schema.ResourceData
 	if err != nil {
 		return diag.FromErr(err)
 	}
-	routes, _, err := session.ClientV2.GetServiceBindingRoutes(serviceID)
+
+	options := client.NewServiceRouteBindingListOptions()
+	options.ServiceInstanceGUIDs = client.Filter{Values: []string{serviceID}}
+
+	routeBindings, err := session.ClientGo.ServiceRouteBindings.ListAll(context.Background(), options)
```

Comment:
```suggestion

	options := client.NewServiceRouteBindingListOptions()
	options.ServiceInstanceGUIDs = client.Filter{Values: []string{serviceID}}
	options.RouteGUIDs = client.Filter{Values: []string{routeID}}

	routeBindings, err := session.ClientGo.ServiceRouteBindings.Single(context.Background(), options)
```

To remove the for loop below

Reply:
I suppose once we switch to use the new V3 id, this entire code will have to change.